generator/internal/auth_decorator_generator.cc

```diff
@@ -68,6 +68,14 @@ Status AuthDecoratorGenerator::GenerateHeader() {
 )""");
 
   for (auto const& method : methods()) {
+    if (IsBidirStreaming(method)) {
+      HeaderPrintMethod(method, __FILE__, __LINE__, R"""(
+  std::unique_ptr<Async$method_name$Stream> Async$method_name$(
+      google::cloud::CompletionQueue& cq,
+      std::unique_ptr<grpc::ClientContext> context) override;
+)""");
+      continue;
+    }
```

Comment:
Because I find it less readable, harder to get maintain (only one of the predicates can match for example, so there are "if X and not Y and not Z" which needs to be changed to "and not W" when you add "W"), and it hides coverage problems (because the "line" is always tested).

Reply:
Makes sense, but looks ugly